docs/source/conf.py

```diff
@@ -89,3 +89,36 @@
     }
 ]
 
+# Generate PDF from pds-api-specification.md URL
+
+from markdown import markdown
+import pdfkit
+import requests
+# import pypandoc
```

Comment:
When code's commented-out, I like to see a comment explaining why. Otherwise, just delete it—with extreme prejudice! 💥